commons/src/main/java/io/hyscale/commons/framework/events/model/InformationEvent.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.commons.framework.events.model;
+
+import java.io.Serializable;
+
+public class InformationEvent<T extends Serializable> extends HyscaleGenericEvent {
+
+    private Level level;
```

Comment:
Do we need this ?

Reply:
With respect to the discussion we have yesterday.

Reply:
It is kept as a utility that can be used later. It would help quicker adaptation of InformationEvent.